### ISSUE TITLE
"Scheduled studies" is unclear label for parents

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#1250]
# TL;DR
The current labels for parents are "All Studies", "Studies happening right now", and "Scheduled studies" -- the label "Scheduled studies" is ambiguous between the meanings "Studies that can be scheduled" (meaning you can make an appointment) and "Studies that have been scheduled" (meaning you already have made an appointment).

# Narrative
In chatting with actual humans narrating their use of the website as part of some light UX testing, multiple people thought they would go to "scheduled studies" to see a list of appointment times they had made, once they started scheduling studies. One option could be the labels "All studies", "Studies you can do now", and "Studies you can schedule for later". Or perhaps "All studies", "Studies you can do now", and "Live studies you schedule for later"

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Decide what the labels should be
- [ ] Update the site
- [ ] Update any documentation that refers to the old labels?
